bbn-test-5/README.md

```diff
@@ -8,17 +8,62 @@ in the covenant committee.
 
 ## Network Parameters
 
-<!-- TODO: sync with devops on the things we should add here.
-Off the top of my head:
-* Chain ID
-* Genesis file location
-* Seed nodes
-* RPC nodes
-* Snapshot
-
-We can fully link to our partners here.
+### Babylon Version
+
+<!-- TODO: Version X.Y.Z -->
+
+### Chain ID
+
+`bbn-test-5`
+
+### Genesis
+
+The genesis file can be retrieved from [here](./network-artifacts/genesis.json).
+
+### Seed nodes
+
+<!-- TODO:
+- Nodes.Guru endpoint
+- Polkachu endpoint
+- In-house endpoint (in case we decide to include)
+-->
+
+### Endpoints
+
+#### RPC
+
+<!-- TODO:
+- Nodes.Guru endpoint
+- Polkachu endpoint
 -->
-...
+
+#### LCD
+
+<!-- TODO:
+- Nodes.Guru endpoint
+- Polkachu endpoint
+-->
+
+#### gRPC
+
+<!-- TODO:
+- Nodes.Guru endpoint
+- Polkachu endpoint
+-->
+
+#### Archive nodes
+
+<!-- TODO:
+- Nodes.Guru endpoint
+- Polkachu endpoint
+-->
+
+### State snapshot
+
+The following snapshot includes state for blocks `1` to `192`, which have
```

Comment:
Is it typical to say the range of block heights?

Also the `which have already been produced` sentence is redundant.

Reply:
I see Polkachu mentions directly the upper limit. We can do the same.